src/actions/vstudio/vs2010_vcxproj.lua

```diff
@@ -1231,7 +1231,15 @@
 
 
 	function m.generateDebugInformation(cfg)
-		m.element("GenerateDebugInformation", nil, tostring(cfg.flags.Symbols ~= nil))
+		if cfg.flags.Symbols and _ACTION >= "vs2015" then
+			if cfg.flags.FastLink then
+				m.element('GenerateDebugInformation', nil, "DebugFastLink")
+			else
+				m.element('GenerateDebugInformation', nil, "Debug")
```

Comment:
Why the change in behaviour in this case? Are `Debug` and `DebugFastLink` the only allowed values in VS 2015?